2005-06-20  Jakub Jelinek  <jakub@redhat.com>

        * sysdeps/unix/sysv/linux/bits/resource.h (RLIMIT_NICE,
        RLIMIT_RTPRIO): Add.
        (RLIMIT_NLIMITS): Adjust.
        * sysdeps/unix/sysv/linux/alpha/bits/resource.h (RLIMIT_NICE,
        RLIMIT_RTPRIO): Add.
        (RLIMIT_NLIMITS): Adjust.
        * sysdeps/unix/sysv/linux/mips/bits/resource.h (RLIMIT_NICE,
        RLIMIT_RTPRIO): Add.
        (RLIMIT_NLIMITS): Adjust.
        * sysdeps/unix/sysv/linux/sparc/bits/resource.h (RLIMIT_NICE,
        RLIMIT_RTPRIO): Add.
        (RLIMIT_NLIMITS): Adjust.


diff --git a/sysdeps/unix/sysv/linux/bits/resource.h b/sysdeps/unix/sysv/linux/bits/resource.h
index ca2c9f0..40b581b 100644
--- a/sysdeps/unix/sysv/linux/bits/resource.h
+++ b/sysdeps/unix/sysv/linux/bits/resource.h
@@ -100,7 +100,13 @@ enum __rlimit_resource
   __RLIMIT_RTPRIO = 14,
 #define RLIMIT_RTPRIO __RLIMIT_RTPRIO
 
-  __RLIMIT_NLIMITS = 15,
+  /* Maximum CPU time that a process scheduled under a real-time
+     scheduling policy may consume without making a blocking
+     system call.  */
+  __RLIMIT_RTTIME = 15,
+#define RLIMIT_RTTIME __RLIMIT_RTTIME
+
+  __RLIMIT_NLIMITS = 16,
   __RLIM_NLIMITS = __RLIMIT_NLIMITS
 #define RLIMIT_NLIMITS __RLIMIT_NLIMITS
 #define RLIM_NLIMITS __RLIM_NLIMITS
